thirdeye/thirdeye-frontend/app/utils/constants.js

```diff
@@ -139,3 +139,24 @@ rules:
 
 `;
 };
+
+export const yamlAlertSettings = `subscriptionGroupName: test_subscription_group
+cron: "0 0/5 * 1/1 * ? *"
+application: "parity-check"
+active: true
+
+detectionConfigIds:
+ - 5773069
+
+fromAddress: thirdeye-dev@linkedin.com
+recipients:
+ to:
+  - "akrai@linkedin.com"
+ cc:
+  - "thirdeye-dev@linkedin.com"
```

Comment:
This has and will potentially have a lot of LinkedIn specific things. I think we should move this out.

Maybe call an endpoint and let the backend return this value. LinkedIn stuff can be easily wrapped in the backend.

Reply:
this is the example template that shown as default in the alert Yaml edit page. To keep this simple, I think a yaml with dummy values (w/o linkedin specific information) would be enough for now.

Reply:
this is the example template that shown as default in the alert Yaml edit page. To keep this simple, I think a yaml with dummy values (w/o linkedin specific information) would be enough for now.

Reply:
this is the example template that shown as default in the alert Yaml edit page. To keep this simple, I think a yaml with dummy values (w/o linkedin specific information) would be enough for now.

Reply:
Yep. Agreed. TOo much work to make it api driven for something so simple as a default. I can make it generic.